schutzbot/deploy.sh

```diff
@@ -71,15 +71,8 @@ retry sudo dnf -y install osbuild-composer-tests
 greenprint "Setting up a directory to hold repository overrides"
 sudo mkdir -p /etc/osbuild-composer/repositories
 
-if [[ -f "rhel-8.json" ]]; then
-    greenprint "Overriding default osbuild-composer rhel-8 sources"
-    sudo cp rhel-8.json /etc/osbuild-composer/repositories/
-fi
-
-if [[ -f "rhel-8-beta.json" ]]; then
-    greenprint "Overriding default osbuild-composer rhel-8-beta sources"
-    sudo cp rhel-8-beta.json /etc/osbuild-composer/repositories/
-fi
+greenprint "Overriding default osbuild-composer sources"
+sudo cp schutzbot/*.json /etc/osbuild-composer/repositories/
```

Comment:
Should be `schutzbot/repositories/*.json`.

Reply:
Indeed :facepalm: thanks